src-plugins/mscAlgorithmPaint/mscAlgorithmPaintToolBox.cpp

```diff
@@ -458,6 +438,9 @@ AlgorithmPaintToolBox::AlgorithmPaintToolBox(QWidget *parent ) :
     connect(reduceBrushSize_shortcut,SIGNAL(activated()),this,SLOT(reduceBrushSize()));
 
     maskHasBeenSaved = false;
+
+    connect(this, SIGNAL(installEventFilterRequest(medViewEventFilter*)),
+            this, SLOT(addViewEventFilter(medViewEventFilter*)));
```

Comment:
The signal/slot system is not needed since both are in the same class.
Just replace `emit install*` by the call of `addViewEventFilter`